src/js/main.js

```diff
@@ -4,7 +4,7 @@ import FontFaceObserver from "fontfaceobserver";
 
 const fontTimeOut = 5000; // In milliseconds
 const numberOfStickers = 7; // How many hero-stickers-0x.svg do we have?
-const minStickableY = 60; // Keep stickers inside viewport top
+const minStickableY = 0; // Make sure sticker can go under the header bar which allows a smoother sticker interaction.
```

Comment:
There doesn't seem to be a need for this variable anymore, right?

Reply:
Indeed thanks. Such stupidity of me to let it live!

Reply:
_KILL IT!!_